web/src/components/ProjectInfo.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+const ProjectWrapper = styled.div`
+  ${tw`container flex flex-col text-brand-9 lg:flex-row mb-24`}
+`;
+
+const StyledHeading = styled.div`
+  ${tw`font-bold mr-16 mb-4 text-5xl lg:text-6xl`}
```

Comment:
Remove the `mr-16` class. It's probably better to rely on flexbox for horizontal alignment.